domains/auth/Auth.API/API/Models/AuthState.cs

```diff
@@ -1,4 +1,7 @@
+using FluentValidation;
+
 namespace API.Models;
+
 public record AuthState
```

Comment:
Why not use the short form declaration?

Reply:
I dont know, i wasnt the one who created this. It would require some refactoring on how we a instantiating this record.
So how do we as a team use the boy scout rule?